src/enums.js

```diff
@@ -79,6 +79,17 @@ export default {
     "2B240303020801010D":    "brainpoolP512r1"
   },
 
+  /** KDF parameters flags
+   * @enum {Integer}
+   * @readonly
+   */
+  kdfFlags: {
```

Comment:
I would suggest to specify forwarding as the use of these flags and state that they're not in the RFC.